pytest/common_lib/shared/__init__.py

```diff
@@ -553,3 +571,211 @@ def new_signer_key(account_id: str) -> Key:
 def generate_signer_keys(account_id: str, num_keys: int) -> list[Key]:
     signer_keys: list[Key] = [new_signer_key(account_id) for _ in range(num_keys)]
     return signer_keys
+
+
+def get_metric_value_for_node(cluster: MpcCluster, metric_name: str, node_id: int):
+    result = cluster.get_int_metric_value_for_node(metric_name, node_id)
+    return result if result is not None else 0
+
+
+def get_node_metrics_all_nodes(cluster: MpcCluster):
+    number_nodes = len(cluster.mpc_nodes)
+
+    network_metrics = [NodeMetrics(0, 0, 0, 0, 0) for _ in range(number_nodes)]
+    for i in range(len(cluster.mpc_nodes)):
+        network_metrics[i].queue_size = get_metric_value_for_node(
+            cluster, "mpc_pending_signatures_queue_size", i
```

Comment:
These helpers are just moved around right? It would be nice to have them in a separate helper module to not pollute `__init__.py`. Also this refactor could be broken out to a separate PR as you said in the meeting today.